clarity/contracts/arkadiko-governance-v4-2.clar

```diff
@@ -40,7 +40,7 @@
 )
 
 (define-data-var governance-shutdown-activated bool false)
-(define-data-var proposal-count uint u6)
+(define-data-var proposal-count uint u0)
```

Comment:
Q: shouldn't we set this to 18? (the ID of the next gov vote)

Reply:
A: Not needed as the FE has logic to handle this properly. Some FE logic was needed anyway, as the proposal ID was not properly set in previous governance contracts.